third_party/2and3/requests/sessions.pyi

```diff
@@ -82,13 +83,13 @@ class Session(SessionRedirectMixin):
                 headers: Optional[MutableMapping[Text, Text]] = ...,
                 cookies: Union[None, RequestsCookieJar, MutableMapping[Text, Text]] = ...,
                 files: Optional[MutableMapping[Text, IO]] = ...,
-                auth: Union[None, Tuple[Text, Text], Callable[[Request], Request]] = ...,
+                auth: Union[None, Tuple[Text, Text], AuthBase, Callable[[Request], Request]] = ...,
                 timeout: Union[None, float, Tuple[float, float]] = ...,
                 allow_redirects: Optional[bool] = ...,
                 proxies: Optional[MutableMapping[Text, Text]] = ...,
                 hooks: Optional[MutableMapping[Text, Callable[[Request], Any]]] = ...,
                 stream: Optional[bool] = ...,
-                verify: Optional[bool] = ...,
+                verify: Union[None, bool, Text] = ...,
```

Comment:
Uh, I'm not sure. I think it gets passed to some normal filesystem functions within requests[1] and then eventually gets used by urllib3 to call the stdlib's ssl.SSLContext.load_verify_locations as the first or second argument[2]. If I believe typeshed[3], it looks like that's just a str. I don't have Python > 3.6 conveniently available, so I haven't tried actually passing PathLike and seeing if it works.

[1] https://github.com/requests/requests/blob/master/requests/adapters.py#L224
[2] https://github.com/shazow/urllib3/blob/master/urllib3/util/ssl_.py#L311
[3] https://github.com/python/typeshed/blob/master/stdlib/3/ssl.pyi#L206